learnJava/exercise2/naloga1.java

```diff
@@ -0,0 +1,22 @@
+import java.util.Scanner;
+
+public class naloga1 {
+    public static void main (String[] args) {
+        Scanner sc = new Scanner (System.in);
+        int n = sc.nextInt();
+
+        if (n > 1) {
+            System.out.print(2);
+        }
+        for (int i = 2; i <= n; i += 2) {
```

Comment:
You already print 2. Why do you need to start a loop with i = 2? Start it with i = 3 and than check for every odd number.